### PR TITLE
[16.0][MIG] sale_order_invoicing_finished_task: Migration to version 16.0

### PR DESCRIPTION
**Pull request by Yadier-Tecnativa**
@Tecnativa TT43245

### CODE REVIEW

**Comment by Yadier-Tecnativa** [comment]
ping @pedrobaeza 
the runboat failed but for other module
``` Module sale_timesheet demo data failed to install, installed without demo data 
odoo.sql_db: bad query: UPDATE "project_project" SET "last_update_id" = 6, "last_update_status" = 'on_track', "write_date" = '2023-05-18T00:18:28.061410'::timestamp, "write_uid" = 1 WHERE id IN (1)
ERROR: insert or update on table "project_project" violates foreign key constraint "project_project_last_update_id_fkey"
DETAIL:  Key (last_update_id)=(6) is not present in table "project_update".

File: sale_order_invoicing_finished_task/models/project.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        self._check_sale_line_state()
+        for task in self:
+            task.invoiceable = not task.invoiceable
+
+    def write(self, vals):
+        if "sale_line_id" in vals:
+            self._check_sale_line_state(vals["sale_line_id"])
+        res = super().write(vals)
+        if "invoiceable" in vals:
+            self.mapped("sale_line_id")._compute_qty_delivered()
```
Comment (hildickethan) [inline sale_order_invoicing_finished_task/models/project.py]: should this not be added to the depends of said compute? instead of forcing it in the write

Comment (Yadier-Tecnativa) [reply sale_order_invoicing_finished_task/models/project.py]: Done

**Review by hildickethan — APPROVED** [review]

**Comment by Yadier-Tecnativa** [comment]
ping @chienandalu , @stefan-tecnativa please review.

the runboat failed:
dropdb: error: database removal failed: ERROR:  database "bb2163fb2-7082-4b60-a6a0-08f04b71e626" is being accessed by other users
DETAIL:  There is 1 other session using the database.

**Pull request merged**